playbooks/manageiq-providers-openstack-test-public-clouds/run.yaml

```diff
@@ -0,0 +1,77 @@
+- hosts: all
+  become: yes
+  roles:
+    - export-cloud-openrc
+  tasks:
+    - name: Run Integration tests of Docker machine against devstack
+      shell:
```

Comment:
can we put these cmd into a role which doing manageiq developer setup?

Reply:
+1